Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -633,70 +648,153 @@ NonnullRefPtr<GUI::Action> HackStudioWidget::create_new_project_action()
     });
 }
 
-void HackStudioWidget::add_new_editor(GUI::Widget& parent)
+NonnullRefPtr<GUI::Action> HackStudioWidget::create_remove_current_editor_tab_widget_action()
+{
+    return GUI::Action::create("Switch to Next Editor Group", { Mod_Alt | Mod_Shift, Key_Backslash }, [this](auto&) {
+        if (m_all_editor_tab_widgets.size() <= 1)
+            return;
+
+        auto tab_widget = m_current_editor_tab_widget;
+        while (!tab_widget->children().is_empty()) {
+            auto* active_wrapper = tab_widget->active_widget();
+            tab_widget->remove_tab(*active_wrapper);
```

Comment:
This causes HackStudio to crash whenever the current tab widget has more than one tab. This might be due to the fact that the function `create_switch_to_next_editor_tab_widget_action()` does not work as intended.

Reply:
All I had to do was remove all but the last tab then, on the last tab, invoke the `on_tab_close_click` action.